modules/end-user-guide/pages/using-a-git-provider-access-token.adoc

```diff
@@ -25,6 +25,11 @@ pass:[<!-- vale RedHat.Spelling = YES -->]
 
 You can create and apply multiple access-token Secrets per Git provider. You must apply each of those Secrets in your user {orch-namespace}.
 
+[NOTE]
+====
+You can configure personal access tokens in *User Preferences* page (`pass:c,a,q[{prod-url}]/dashboard/#/user-preferences?tab=personal-access-tokens`) of your {prod-short} dashboard.
```

Comment:
```suggestion
You can configure personal access tokens on *User Preferences* page of your {prod-short} dashboard: `pass:c,a,q[{prod-url}]/dashboard/#/user-preferences?tab=personal-access-tokens`.
```

Reply:
Looks good!

Reply:
@deerskindoll shouldn't we add the e.g. `on the  *User Preferences* page`

Reply:
@akurinnoy I think we should move this to the beginning of the article smth. like 

 ```
OAuth for GitHub, GitLab, Bitbucket, or Microsoft Azure Repos needs to be xref:administration-guide:configuring-oauth-for-git-providers.adoc[configured by the administrator] of your organization's {prod-short} instance. If your administrator could not configure it for {prod-short} users, the workaround is for you to use a personal access token. You can configure personal access tokens on the *User Preferences* page of your {prod-short} dashboard: `pass:c,a,q[{prod-url}]/dashboard/#/user-preferences?tab=personal-access-tokens`, or apply it manually as a {kubernetes} Secret in the namespace.
```

Reply:
@ibuziuk you're absolutely right, `the page` is correct.